examples/basic/packages/tsconfig/base.json

```diff
@@ -2,20 +2,18 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "Default",
   "compilerOptions": {
-    "composite": false,
-    "declaration": true,
-    "declarationMap": true,
     "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "inlineSources": false,
-    "isolatedModules": true,
-    "moduleResolution": "node",
-    "noUnusedLocals": false,
-    "noUnusedParameters": false,
-    "preserveWatchOutput": true,
     "skipLibCheck": true,
+    "target": "es2022",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "moduleDetection": "force",
     "strict": true,
-    "strictNullChecks": true
+    "noUncheckedIndexedAccess": true,
+    "moduleResolution": "Bundler",
```

Comment:
This line is failing CI

> web:build: FatalError: ../../packages/tsconfig/nextjs.json(15,25): error TS6046: Argument for '--moduleResolution' option must be: 'node', 'classic', 'node16', 'nodenext'.

I assume our TS version needs updating (and maybe `bundler` needs to be lowercase?)

Reply:
TS version needs updating - where is that handled?

Reply:
This PR should do what you need https://github.com/vercel/turbo/pull/6338

Reply:
Perfect